common/ideas/00_basic_ideas.txt

```diff
@@ -254,18 +254,19 @@ administrative_ideas = {
 		core_creation = -0.25
 	}
 	improved travel = {
-		local_friendly_movement_speed = 0.2
+		movement_speed = -0.15
```

Comment:
I believe reductions to movement speed will be interpreted by the game as a debuff rather than a buff. This is something that springs up in a couple of modifiers (ex: technology cost = -0.05 will actually increase the cost you need to pay)